modules/stac-server/lambda/waiting_for_opensearch/main.py

```diff
@@ -16,17 +16,22 @@ def lambda_handler(event, context):
 def wait_for_opensearch_serverless_collection(collection):
     collection_active = False
     while not collection_active:
```

Comment:
I usually like to do something like

```python
total_slept = 0
while not collection_active and total_slept <= 300:
  ...
  
  time.sleep(5)
  total_slept += 5
```

and then error out with a good message if it's not available after all the timeout